frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -164,7 +157,7 @@ const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
               <div>
                 <div>
                   <p className="co-clone-pvc-modal__pvc-details">Access Mode</p>
-                  <p>{accessModeLabels[resource.spec.accessModes]}</p>
+                  <p>{accessModes.title}</p>
```

Comment:
what happen when accessMode is undefined i.e. the search accessMode is not present?

Reply:
ack, I have handled it